src/main/resources/templates/java/test/gradle/projectTemplate/build.gradle

```
@@ -0,0 +1,160 @@
+plugins {
+    id 'java'
+    // %static-code-analysis-start%
+    id 'checkstyle'
+    id 'pmd'
+    id 'com.github.spotbugs' version '5.0.6'
+    // %static-code-analysis-stop%
+}
+
+repositories {
+    mavenCentral()
+    mavenLocal()
+}
+
+dependencies {
+    implementation 'de.tum.in.ase:artemis-java-test-sandbox:1.9.0'
```

Since Ares is only used in the tests, I think `testImplementation` should be enough here. Related https://github.com/ls1intum/Ares/issues/197
```suggestion
    testImplementation 'de.tum.in.ase:artemis-java-test-sandbox:1.9.0'
```